### ISSUE TITLE
Yoga 7 16 (16ARP8) ?

### ISSUE THREAD

**Issue by johnstonesnow** [tomsom/yoga-linux#54]
Sorry, I think I posted this on another guthub thread by accident, I meant to post it here...

I have just bought a Lenovo Yoga 7 16" (16ARP8) and trying to run Fedora 38/39 on it.
I notice a few issues (bad camera quality, speakers terrible, silent below 50% volume, audible at 100% if in a silent room, tablet mode seems to work ok but top row of keyboard for some reason is still active after folding screen back to tablet form.

I am new to Linux and new to Github, really struggle to understand even just how to use it, but I can use CLI commands as and when I am forced to!

Just wanted to ask if the patches/tricks shown on this thread have ANY chance of working on my machine? If not, I won't mess my system up with commands I won't know how to undo! But if there is a chance, I may give some of the suggestions a go.

Grateful for any thoughts on whether it's worth a shot. Thanks, some great work here, if only there was similar specificallly for my Yoga 7! (One day maybe!)

**Comment by stuarthayhurst** [comment]
Most of the fixes here have been aimed at upstream, so should just work out of the box on new distros. If you've got an issue with tablet mode that doesn't happen on Windows, perhaps this could be solved with a quirk in the `lenovo-ymc` or `ideapad-laptop` platform drivers?

You might have some luck with the speakers in #3, I think another user with your laptop had a similar issue.

**Comment by johnstonesnow** [comment]
Could you please explain what "aimed at upstream" means? Do you mean your edits are now bundled into the OS via updates?
thanks

**Comment by stuarthayhurst** [comment]
Yes, most of the fixes here get applied to the projects, then the new versions eventually make it to the distro you use, so you'll get them as a regular update at some point.

Since you have a slightly different model to the rest of us, not all the fixes will apply to you, but the shared hardware will benefit from the work done.

**Comment by soyersoyer** [comment]
https://bugzilla.kernel.org/show_bug.cgi?id=216925
